src/controllers/users.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const usersService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = usersService.getAll();
+
+  res.send(users);
+};
+
+const findUserById = (req, res) => {
```

Comment:
The same as for expenses + be consistent, all other methods have a simple name (getAll, add ....)